sunderland/theme.json

```diff
@@ -1251,17 +952,17 @@
 			}
 		},
 		"spacing": {
-			"blockGap": "1.5rem",
+			"blockGap": "2rem",
 			"padding": {
 				"bottom": "0rem",
-				"left": "0rem",
-				"right": "0rem",
+				"left": "5vw",
+				"right": "5vw",
 				"top": "0rem"
 			}
 		},
 		"typography": {
```

Comment:
I could not change this myself in theme.json but we should neither use or bundle the font Helvetica with the theme, we should just specify Helvetica as a system font, just like the theme "Rainfall" does this.

Reply:
I've removed Helvetica as a bundled font here: https://github.com/Automattic/themes/pull/7812/commits/4535d77b5955797767c93382bf4565515ca18167